container/shim/src/index.js

```diff
@@ -118,6 +118,11 @@ const handleCID = asyncHandler(async (req, res) => {
 
   debug(`Cache miss for ${req.path}`);
 
+  if ((req.headers["x-fetcher"] || "").indexOf("bifrost-gateway") > -1) {
```

Comment:
```diff
-   if ((req.headers["x-fetcher"] || "").indexOf("bifrost-gateway") > -1) {
+   if ((req.headers["x-fetcher"] || "").includes("bifrost-gateway")) {
```

Bystander comment: This should be easier to read

Reply:
😅  my JS is a couple years out of date

Reply:
```js
if (req.headers["x-fetcher"]?.includes("bifrost-gateway"))`
```

prolly the most l33t hacker version

Reply:
Nice!